Source/UI/BillingForm/View/BillingFormHeaderCell.swift

```diff
@@ -45,14 +45,6 @@ final class BillingFormHeaderCell: UIView {
         headerLabel.update(with: style.headerLabel)
     }
 
-    @objc private func doneAction() {
-        delegate?.doneButtonIsPressed()
-    }
-
-    @objc private func cancelAction() {
-        delegate?.cancelButtonIsPressed()
-    }
-
```

Comment:
please share details for deleting these delegates?

Reply:
This is dead code which was not called anywhere.

Reply:
How come? if you did a little search you will find it in 
```
extension BillingFormViewController: BillingFormHeaderCellDelegate {

    func doneButtonIsPressed() {
        delegate?.doneButtonIsPressed(sender: self)
    }

    func cancelButtonIsPressed() {
        delegate?.cancelButtonIsPressed(sender: self)
    }
}```

Reply:
Please see the code you referred to and the code being deleted

Reply:
??

Reply:
The code being deleted is `doneAction` and `cancelAction`. You're looking at different things